### PR TITLE
Simplify validation tests

### PR DESCRIPTION
**Pull request by edaena**

### CODE REVIEW

File: src/commands/hld/pipeline.test.ts [hunk 1]
```diff
@@ -71,129 +61,33 @@ describe("validate pipeline config", () => {
   it("config is valid", () => {
     expect(
       isValidConfig(
```
Comment (evanlouie) [inline src/commands/hld/pipeline.test.ts]: you can simplify this to `isValidConfig.apply(undefined, configValues)`

File: src/commands/project/pipeline.test.ts [hunk 2]
```diff
@@ -24,139 +24,47 @@ afterAll(() => {
...
+    "repoName",
+    "https:/repoulr",
+    "https://hldurl",
+    "https://buildscript",
+    "af8e99c1234ef93e8c4365b1dc9bd8d9ba987d3"
+  ];
+
   it("config is valid", () => {
     expect(
       isValidConfig(
```
Comment (evanlouie) [inline src/commands/project/pipeline.test.ts]: you can simplify this to `isValidConfig.apply(undefined, configValues)`

File: src/commands/service/create.test.ts [hunk 3]
```diff
@@ -25,220 +25,56 @@ afterAll(() => {
...
+    "test/packages",
+    "test-maintainer",
+    "test@maintainer.com",
+    true,
+    "testVariableGroup"
+  ];
+
   it("config is valid", () => {
     expect(
       isValidConfig(
```
Comment (evanlouie) [inline src/commands/service/create.test.ts]: you can simplify this to `isValidConfig.apply(undefined, configValues)`

File: src/commands/service/create.test.ts [hunk 4]
```diff
@@ -25,220 +25,56 @@ afterAll(() => {
...
-        "testService",
-        "test/packages",
-        "test-maintainer",
-        "test@maintainer.com",
-        true,
-        undefined
-      )
-    ).toBe(false);
+  it("undefined parameters", () => {
+    for (let i = 0; i < configValues.length; i++) {
```
Comment (evanlouie) [inline src/commands/service/create.test.ts]: Can be simplified:

```typescript
for (const i of configValues.keys()) {
  const configValuesWithInvalidValue = configValues.map((value, j) =>
    i === j ? undefined : value
  );
  expect(isValidConfig.apply(undefined, configValuesWithInvalidValue)).toBe(
    false
  );
}
```

File: src/commands/project/pipeline.test.ts [hunk 5]
```diff
@@ -24,139 +24,47 @@ afterAll(() => {
...
-        "testPipeline",
-        "repoName",
-        "https:/repoulr",
-        "https://hldurl",
-        "https://buildscript",
-        undefined
-      )
-    ).toBe(false);
+  it("undefined values", () => {
+    for (let i = 0; i < configValues.length; i++) {
```
Comment (evanlouie) [inline src/commands/project/pipeline.test.ts]: Can be simplified:

```typescript
for (const i of configValues.keys()) {
  const configValuesWithInvalidValue = configValues.map((value, j) =>
    i === j ? undefined : value
  );
  expect(isValidConfig.apply(undefined, configValuesWithInvalidValue)).toBe(
    false
  );
}
```

File: src/commands/hld/pipeline.test.ts [hunk 6]
```diff
@@ -71,129 +61,33 @@ describe("validate pipeline config", () => {
...
-        "testPipeline",
-        "https://manifestulr",
-        "testHld",
-        "https://hldurl",
-        "https://buildscript",
-        4
-      )
-    ).toBe(false);
+  it("undefined values", () => {
+    for (let i = 0; i < configValues.length; i++) {
```
Comment (evanlouie) [inline src/commands/hld/pipeline.test.ts]: Can be simplified:

```typescript
for (const i of configValues.keys()) {
  const configValuesWithInvalidValue = configValues.map((value, j) =>
    i === j ? undefined : value
  );
  expect(isValidConfig.apply(undefined, configValuesWithInvalidValue)).toBe(
    false
  );
}
```

**Review by evanlouie — CHANGES_REQUESTED** [review]
like the simplifications.
Couple of changes to make them more resilient to adding of more parameters

**Review by evanlouie — APPROVED** [review]

**Review by samiyaakhtar — APPROVED** [review]

**Pull request merged**